etc/aqd.conf.lal

```diff
@@ -0,0 +1,150 @@
+# This config file is for running the broker outside of MS where no afs available.
```

Comment:
What is this :) ?

Reply:
Another mistake of mine, should not be committed....